src/elements/picker/extras/entity-picker-results/EntityPickerResults.js

```diff
@@ -76,6 +76,8 @@ export class EntityPickerResults extends PickerResults {
                     return 'lead';
```

Comment:
Why is this code in here? Its already in master. Did you make this off a stale branch without getting master first?